src/EmbeddedSonarAnalyzer.props

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <EmbeddedSonarAnalyzerVersion>8.19.0.28253</EmbeddedSonarAnalyzerVersion>
     <EmbeddedSonarCFamilyAnalyzerVersion>6.17.0.27551</EmbeddedSonarCFamilyAnalyzerVersion>
-    <EmbeddedSonarJSAnalyzerVersion>7.4.0.15161</EmbeddedSonarJSAnalyzerVersion>
+    <EmbeddedSonarJSAnalyzerVersion>7.4.0.15221</EmbeddedSonarJSAnalyzerVersion>
```

Comment:
New version with the `defaultParams` metadata. The rest of the data is the same.